src/get/progress.rs

```diff
@@ -183,3 +183,90 @@ impl TransferState {
         }
     }
 }
+
+/// Progress updates for the get operation.
+#[derive(Debug, Clone, Serialize, Deserialize)]
+pub enum DownloadProgress {
```

Comment:
can we normalize `get`, `download` and `fetch`? they sound like the same thing for most users

Reply:
Normalize to what? get, download or fetch? I guess the most descriptive is download, given that it is a downloader. 

But then get is more accurate for the case where the stuff is already local.

Reply:
I think `fetch` would work reasonably well (given that is how the browser calls this too)  and then adding suffixes if you talk about expliclit local or remote fetching

- `fetch` - could be remote or local
- `fetch_local` only reads local data
- `fetch_remote` only reads data from other peers